src/components/cylc/Drawer.vue

```diff
@@ -57,30 +55,16 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
           class="v-list-item"
       >
         <v-list-item-action>
-          <v-icon>mdi-graphql</v-icon>
+          <v-icon>{{ svgPaths.graphql }}</v-icon>
         </v-list-item-action>
         <v-list-item-title>GraphiQL</v-list-item-title>
       </v-list-item>
       <v-divider />
       <v-subheader>Workflows</v-subheader>
-      <v-list-item
-        v-for="(link, index) in viewLinks"
-        :key="index+link.text"
-        :to="link.to"
-        active-class="primary grey--text text--darken-3"
-        class="v-list-item"
-      >
-        <v-list-item-action>
-          <v-icon>{{ link.icon }}</v-icon>
-        </v-list-item-action>
-        <v-list-item-title
-          v-text="link.text"
-        />
-      </v-list-item>
```

Comment:
What's going on here?

Reply:
Unused code. We used to have more links in the drawer, before 0.1, before matching the design sketch.

This code is from the tim creative design, where it would add views links dynsmically.